src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -524,59 +525,60 @@ export default function Swap({
         onDismiss={closeModals}
         operationType={operationType}
       />
-
-      <NetworkAlert />
-      <StyledAppBody className={className}>
-        <SwapHeader allowedSlippage={allowedSlippage} />
-        <Wrapper id="swap-page" className={isExpertMode ? 'expertMode' : ''}>
-          <ConfirmSwapModal
-            isOpen={showConfirm}
-            trade={trade}
-            originalTrade={tradeToConfirm}
-            onAcceptChanges={handleAcceptChanges}
-            attemptingTxn={attemptingTxn}
-            txHash={txHash}
-            recipient={recipient}
-            allowedSlippage={allowedSlippage}
-            onConfirm={handleSwap}
-            swapErrorMessage={swapErrorMessage}
-            onDismiss={handleConfirmDismiss}
-          />
-
-          <AutoColumn gap={'md'}>
-            <div style={{ display: 'relative' }}>
-              <CurrencyInputPanel
-                // label={
-                //   independentField === Field.OUTPUT && !showWrap ? <Trans>From (at most)</Trans> : <Trans>From</Trans>
-                // }
-                label={
-                  exactInLabel && (
-                    <FeeInformationTooltip
-                      label={exactInLabel}
-                      trade={trade}
-                      showHelper={independentField === Field.OUTPUT}
-                      amountBeforeFees={amountBeforeFees}
-                      amountAfterFees={formatSmart(trade?.inputAmountWithFee, AMOUNT_PRECISION)}
-                      type="From"
-                      feeAmount={formatSmart(trade?.fee?.feeAsCurrency, AMOUNT_PRECISION)}
-                      allowsOffchainSigning={allowsOffchainSigning}
-                      fiatValue={fiatValueInput}
-                    />
-                  )
-                }
-                value={formattedAmounts[Field.INPUT]}
-                showMaxButton={showMaxButton}
-                currency={currencies[Field.INPUT]}
-                onUserInput={handleTypeInput}
-                onMax={handleMaxInput}
-                fiatValue={fiatValueInput ?? undefined}
-                onCurrencySelect={handleInputSelect}
-                otherCurrency={currencies[Field.OUTPUT]}
-                showCommonBases
-                id="swap-currency-input"
-              />
-              {/* UNI ARROW SWITCHER */}
-              {/* 
+      <Container>
```

Comment:
Since this is a "mod" file, we ideally keep the changes to a minimum
I see what why you need this change and it does make sense, just wondering how to make it in a less intrusive way so our uni-swap-merger-buddies have less hair to pull on next update.

What about you add the `<Container>` and `<AffiliateStatusCheck>` components to the ../index.tsx file and there you "wrap" this component, something like:

```ts
// index.tsx
import { Swap as SwapMod } from 'pages/Swap/SwapMod'

export function Swap(props) {
  return (
    <Container>
      <SwapMod props={props}/>
    </Container>
  )
}
```

Code not tested, not sure it will work, just an idea

Thoughts, uni-swap-merge-experts? @W3stside @nenadV91

Reply:
Done! @alfetopito

Reply:
Nice! Great job